crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3895,9 +3895,43 @@ mod tests {
         assert!(std::mem::size_of::<StmtClassDef>() <= 104);
         assert!(std::mem::size_of::<StmtTry>() <= 112);
         // 80 for Rustc < 1.76
-        assert!(matches!(std::mem::size_of::<Expr>(), 72 | 80));
+        assert!(matches!(std::mem::size_of::<Expr>(), 56 | 64));
         // 96 for Rustc < 1.76
         assert!(matches!(std::mem::size_of::<Pattern>(), 88 | 96));
         assert!(std::mem::size_of::<Mod>() <= 32);
+
+        // static_assertions::assert_eq_size!(Expr, [u8; 64]);
+        // static_assertions::assert_eq_size!(ExprAttribute, [u8; 56]);
+        // static_assertions::assert_eq_size!(ExprAwait, [u8; 16]);
+        // static_assertions::assert_eq_size!(ExprBinOp, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprBoolOp, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprBooleanLiteral, [u8; 12]);
+        // static_assertions::assert_eq_size!(ExprBytesLiteral, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprCall, [u8; 56]);
+        // static_assertions::assert_eq_size!(ExprCompare, [u8; 48]);
+        // static_assertions::assert_eq_size!(ExprDict, [u8; 56]);
+        // static_assertions::assert_eq_size!(ExprDictComp, [u8; 48]);
+        // static_assertions::assert_eq_size!(ExprEllipsisLiteral, [u8; 8]);
+        // static_assertions::assert_eq_size!(ExprFString, [u8; 48]);
+        // static_assertions::assert_eq_size!(ExprGeneratorExp, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprIfExp, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprIpyEscapeCommand, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprLambda, [u8; 24]);
+        // static_assertions::assert_eq_size!(ExprList, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprListComp, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprName, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprNamedExpr, [u8; 24]);
+        // static_assertions::assert_eq_size!(ExprNoneLiteral, [u8; 8]);
+        // static_assertions::assert_eq_size!(ExprNumberLiteral, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprSet, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprSetComp, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprSlice, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprStarred, [u8; 24]);
+        // static_assertions::assert_eq_size!(ExprStringLiteral, [u8; 48]);
+        // static_assertions::assert_eq_size!(ExprSubscript, [u8; 32]);
+        // static_assertions::assert_eq_size!(ExprTuple, [u8; 40]);
+        // static_assertions::assert_eq_size!(ExprUnaryOp, [u8; 24]);
+        // static_assertions::assert_eq_size!(ExprYield, [u8; 16]);
+        // static_assertions::assert_eq_size!(ExprYieldFrom, [u8; 16]);
```

Comment:
@MichaReiser - I missed your comment when we talked in-person -- did you want me to include these as `assert!`?

Reply:
Ah sorry. I don't mind keeping them. I do find them useful when analyzing the type's size in the future.